src/machines/java/com/enderio/machines/common/init/MachineBlocks.java

```diff
@@ -200,6 +200,17 @@ public class MachineBlocks {
     public static final BlockEntry<ProgressMachineBlock> DRAIN = progressMachine("drain", () -> MachineBlockEntities.DRAIN)
         .register();
 
+    public static final BlockEntry<MachineBlock> XP_OBELISK = REGISTRATE
```

Comment:
Custom .lang to capitalise "XP".

Could we also do this on the XP Juice as its lowercase too - cheers :)